### DISCUSSION TITLE
New version 0.16

### DISCUSSION THREAD

**Discussion by dl2ki** [foldynl/QLog#132] [General]
Hi,

thank you very much for the new version. The addition for manual QSO input works very well. Very good work.

Otherwise, QLog works smoothly here and so far without any noticeable bugs, so suggestions for improvements are sure to become fewer. If the extension according to "Issue #60" still comes sometime, QLog is perfect for me.

73, Wolfgang
DL2KI

**Comment by foldynl** [comment]
Hi Wolfgang,

thank you very much for your feedback.

Currently, I am planing the next release. I'll take a look at it.
